### ISSUE TITLE
Script times out

### ISSUE THREAD

**Issue by Redguy1959** [radius314/bmlt-geo-patch#1]
This script times out. I assume that is a server setting. Correct me if I'm wrong, but it looks like this script geocodes the entire database? Is there a way to give it just one ASC at a time so as to avoid the timeout?

**Comment by dgershman** [comment]
Yes you could modify this line (https://github.com/bmlt-enabled/bmlt-geo-patch/blob/master/index.php#L7) and add a more specific query.  For example to pick a particular area.

$root_server . "/client_interface/json/?switcher=GetSearchResults&services[]=2.

Use the semantic workshop to get the exact ID of the service body.
